### PR TITLE
Improvement modules bhima

### LINKED ISSUE #6109: Basic IMPROVEMENT IN BHIMA
- Added the bar to display the total as well as the number of data configuring in the Payroll Elements configuration interface with index
- Improvement of the patient configuration registration page in workers
- Improvement of employee update form
- Add a link between the updated page of patients to the registration page

### PR DESCRIPTION
**Pull request by lomamech**
- Improve Staffign Indice grice display footer cell class
- Adding link between Patient Edit page and Patient Information
- Prevent re-registering a patient who is already employed
- Added the status of the employee in the interface

closes #6109

### CODE REVIEW

**Comment by lomamech** [comment]
I had noticed an error when I was working with the VANGA server but I cannot reproduce it locally even with their databases, when trying to transform a patient into an employee, it sometimes happened that the components location was partially loaded and ask that we refresh the page to have the data in full

File: client/src/modules/patients/record/patient-record.html [hunk 1]
```diff
@@ -105,13 +105,21 @@
...
-                <a
+                <a ng-if="!PatientRecordCtrl.patient.employee_uuid"
                   data-method="convert-to-employee"
                   ui-sref="patientAsEmployee({ uuid : PatientRecordCtrl.patient.uuid })"
                   class="btn btn-default btn-block">
                   <span class="fa fa-street-view"></span>
                   <span translate>FORM.LABELS.REGISTER_EMPLOYEE_PATIENT</span>
                 </a>
+
+                <span ng-if="PatientRecordCtrl.patient.employee_uuid && !PatientRecordCtrl.patient.locked" class="btn btn-success btn-block" translate>
```
Comment (jniles) [inline client/src/modules/patients/record/patient-record.html]: I would prefer if this didn't look like a button.  I tried to click it and was confused why nothing happened.  Can you just make it a `<p class="text-success">`?

File: client/src/modules/patients/record/patient-record.html [hunk 2]
```diff
@@ -105,13 +105,21 @@
...
+                  EMPLOYEE.ACTIVE_EMPLOYEE
+                </span>
+
+                <span ng-if="PatientRecordCtrl.patient.employee_uuid && PatientRecordCtrl.patient.locked" class="btn btn-warning btn-block" translate>
```
Comment (jniles) [inline client/src/modules/patients/record/patient-record.html]: Same comment about the button.

File: client/src/modules/payroll/staffing_indice/staffingIndice.html [hunk 3]
```diff
@@ -69,8 +69,15 @@
 
 <div class="flex-content">
   <div class="container-fluid">
-    <div id="staffing-indice-grid" class="grid-full-height" ui-grid="StaffingIndiceCtrl.gridOptions" ui-grid-auto-resize
-      ui-grid-resize-columns ui-grid-move-columns>
+    <div
+      id="staffing-indice-grid"
+      class="grid-full-height-with-filters"
```
Comment (jniles) [inline client/src/modules/payroll/staffing_indice/staffingIndice.html:74]: 👍

**Review by jniles — COMMENTED** [review]
Just one comment about the confusing buttons that aren't buttons.  Otherwise, LGTM.

**Comment by lomamech** [comment]
image
This is the new looking

File: client/src/modules/patients/record/patient-record.html [hunk 4]
```diff
@@ -105,13 +105,21 @@
...
-                <a
+                <a ng-if="!PatientRecordCtrl.patient.employee_uuid"
                   data-method="convert-to-employee"
                   ui-sref="patientAsEmployee({ uuid : PatientRecordCtrl.patient.uuid })"
                   class="btn btn-default btn-block">
                   <span class="fa fa-street-view"></span>
                   <span translate>FORM.LABELS.REGISTER_EMPLOYEE_PATIENT</span>
                 </a>
+
+                <p ng-if="PatientRecordCtrl.patient.employee_uuid && !PatientRecordCtrl.patient.locked" class="badge badge-success" translate>
```
Comment (jniles) [inline client/src/modules/patients/record/patient-record.html]: Much better!

**Review by jniles — APPROVED** [review]
LGTM.

bors r+

**Pull request merged**